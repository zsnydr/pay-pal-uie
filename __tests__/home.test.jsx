import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Home from '../public/home/home';

describe('Home (Snapshot)', () => {
  it('Home renders home page', () => {
    const component = renderer.create(<Home />);
    const home = component.toJSON();
    expect(home).toMatchSnapshot();
  });
});

describe('Home', () => {
  let home;
  let history;
  beforeEach(() => {
    history = [];
    home = mount(<Home history={history} />);
  });

  xit('Button click calls clickHandler', () => {
    const button = home.find('button').first();
    button.simulate('click');
    expect(home.clickHandler).toExist();
  });
});
