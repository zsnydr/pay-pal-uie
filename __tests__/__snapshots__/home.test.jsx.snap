// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home (Snapshot) Home renders home page 1`] = `
<div
  className="home"
>
  <div
    className="header"
  >
    What are we Doing?
  </div>
  <div
    className="home-button-container"
  >
    <button
      className="home-button"
      name="send"
      onClick={[Function]}
    >
      Send Money
    </button>
    <button
      className="home-button"
      name="history"
      onClick={[Function]}
    >
      View Transaction History
    </button>
  </div>
  <div
    className="footer"
  />
</div>
`;
